src/test/java/nextstep/subway/line/LineAcceptanceTest.java

```diff
@@ -11,22 +26,30 @@
     void createLine() {
         // when
         // 지하철_노선_생성_요청
+        ExtractableResponse<Response> response = createLineResponse("신분당선", "red");
 
         // then
         // 지하철_노선_생성됨
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.CREATED.value());
+        assertThat(response.header("Location")).isNotBlank();
```

Comment:
```suggestion
        // when
        ExtractableResponse<Response> response = 지하철_노선_생성_요청(request);

        // then
        지하철_노선_생성됨(response);
```
이러한 형태로 분리해서 작성해보세요!

Reply:
then과 관련된 메소드는 현재 Test 클래스에서 작성하고,
when과 관련된 메소드는 `LineStpes` 클래스로 분리해서 작성하면 추후에 코드를 재활용하기 수월하답니다!

Reply:
그리고 이렇게 분리하는 이유 중 하나는, 인수 테스트 자체가 이것 저것 작성해야할 코드가 많은 편입니다. 그래서 테스트 자체에 더 집중하기 위함이라고 생각해주세요!

Reply:
생각해보니 junit4 사용할 때는 테스트 메소드명을 한글로 썼던 적도 있는데 왜 내부에서 사용할 땐 안 써먹었는지 모르겠네요 ㅎㅎ...
포비님도 한글로 써도 된다고 하셨는데 저도 모르게 쓸데없는 고집이 생겼던 것 같습니다 😞